paper/main.tex

```diff
@@ -718,19 +718,6 @@ \section*{Discussion}
 unlikely to reflect non-stimulus-driven dynamics that are unlikely to
 be similar across individuals.
 
-One limitation of our approach relates to how noise propagates in our
-estimation procedure.  Specifically, our procedure for estimating
-high-order dynamic correlations depends on estimates of lower-order
-dynamic correlations.  This means that our measures of which
-higher-order patterns are reliable and stable across experimental
-conditions are partially confounded with the stability of lower-order
-patterns.  Prior work suggests that the stability of what we refer to
-here as first-order dynamics likely varies across the experimental
-conditions we examined~\citep{SimoEtal16}.  Therefore a caveat to our
-claim that richer stimuli evoke more stable higher-order dynamics is
-that our approach assumes that those high-order dynamics reflect
-relations or interactions between lower-order features.
-
```

Comment:
can you add this back?

Reply:
ok I added this entire section back (I think that's what you were asking for???)

One limitation of our approach relates to how noise propagates in our
estimation procedure.  Specifically, our procedure for estimating
high-order dynamic correlations depends on estimates of lower-order
dynamic correlations.  This means that our measures of which
higher-order patterns are reliable and stable across experimental
conditions are partially confounded with the stability of lower-order
patterns.  Prior work suggests that the stability of what we refer to
here as first-order dynamics likely varies across the experimental
conditions we examined~\citep{SimoEtal16}.  Therefore a caveat to our
claim that richer stimuli evoke more stable higher-order dynamics is
that our approach assumes that those high-order dynamics reflect
relations or interactions between lower-order features.